tests/composer/composer-symfony3-max.json

```
@@ -0,0 +1,55 @@
+{
+    "name": "propel/propel",
+    "type": "library",
+    "description": "Propel2 is an open-source Object-Relational Mapping (ORM) for PHP 5.5 and up.",
+    "keywords": [
+        "ORM",
+        "persistence",
+        "Active Record"
+    ],
+    "homepage": "http://www.propelorm.org/",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "William Durand",
+            "email": "william.durand1@gmail.com"
+        }
+    ],
+    "require": {
+        "php": ">=7.1",
+        "symfony/yaml": "^3.0.0",
+        "symfony/config": "^3.0.0",
+        "symfony/console": "^3.0.0",
+        "symfony/filesystem": "^3.0.0",
+        "symfony/finder": "^3.0.0",
+        "symfony/translation": "^3.0.0",
+        "symfony/validator": "~3.1.7||^3.2.4",
```

Might need to become 
```suggestion
        "symfony/validator": "^3.3.0",
```